german/schwimmen/gefahr/wald/flucht.md

```diff
@@ -0,0 +1,2 @@
+Sie rennen in Panik los. Dabei stolpern Sie über das dichte Wurzelwerk und fallen hin.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.